### PR TITLE
Add a brightness characteristics only when needed

### PR DESCRIPTION
**Pull request by shmuelzon**

### CODE REVIEW

**Comment by shmuelzon** [comment]
https://github.com/home-assistant/home-assistant/pull/2828 was merged, so now we can properly know what the Light supports

**Comment by maddox** [comment]
This is great! Let's wait till that lands on a release and we can merge this and push out a release.

File: accessories/light.js [hunk 1]
```diff
@@ -25,6 +25,20 @@ function HomeAssistantLight(log, data, client) {
 }
 
 HomeAssistantLight.prototype = {
+  features: Object.freeze({
```
Comment (maddox) [inline accessories/light.js]: Thanks for setting up for the future.

**Comment by shmuelzon** [comment]
OK, no problem.

**Comment by robbiet480** [comment]
I'm going to merge this now since we have a check in place for `supported_features` existence.

**Pull request merged**